modules/components/pages/inputs/postgres_cdc.adoc

```diff
@@ -158,6 +158,44 @@ For full visibility of all data updates, you can also choose to stream https://w
 ALTER TABLE large_data REPLICA IDENTITY FULL;
 ```
 
+== Data mappings
+
+The following table shows how selected PostgreSQL data types are mapped to data types supported in Redpanda Connect. All other data types are mapped to string values.
+
+|===
+| PostgreSQL data type | Mapped to
```

Comment:
`Mapped to` should maybe be `Bloblang value` like we did in the Snowflake Connector (except this is the other direction because it's an input not an output)?

Reply:
Updated.